src/utils/navigasjon.ts

```diff
@@ -22,6 +22,12 @@ const onClickLink = (event: any, sti: string) => {
 const gaaTilDigitalSoknad = (kommuneId?: string): void => {
     const query = kommuneId !== undefined ? "?kommuneId=" + kommuneId : "";
     let soknadUrl: string = "/sosialhjelp/soknad/informasjon" + query;
+    if (window.location.origin.indexOf(".dev-nav.no") >= 0) {
```

Comment:
Bruker vi **_dev-nav_** i dag? 
Hvis ikke vil vel denne i fremtiden ta over for q1 og q0-miljøene? 
Isåfall tenker jeg at vi fortsatt bruker de som integrerte miljøer og ikke mockede så man skal bli sendt til /sosialhjelp/soknad/informasjon og ikke /mock-login?

Reply:
dev-nav.no var det første stedet vi fikk dette ut på GCP. Det er en større jobb å få satt dette opp med integrasjon og Oracle->>Postgres. Når vi gjør dette så kan vi også fikse denne URLen.